### DISCUSSION TITLE
Refresh (git) gutter command

### DISCUSSION THREAD

**Discussion by JurajLieskovsky** [helix-editor/helix#6827] [Ideas]
I don't mind that my files do not automatically reload when they are changed outside of helix but something that bothers me a bit is that after committing changes to git I have to reload the whole file for my gutter to update.

I am not certain that helix compares changes to the last commit, but if it does a `:refresh-gutter` command would be great.

At the moment I run `:w`, `:reload` whenever I am not sure my gutter is up to date, but it's a bit of a hassle.

---

If the gutter updated automatically every time helix is re-focused (similarly to how you can "auto-save" when focus moves away from it), that would be pretty amazing.

**Comment by grv07** [comment]
I have map `Alt + r -> :reload-all` for this

**Comment by kirawi** [comment]
This would still require file watching to know if a commit was made.
